src/ClaimService.Data.Provider.MsSql.Ef/ClaimServiceDbContext.cs

```diff
@@ -1,10 +1,21 @@
 ﻿using System.Threading.Tasks;
+using LT.DigitalOffice.ClaimService.Models.Db;
 using Microsoft.EntityFrameworkCore;
 
 namespace ClaimService.Data.Provider.MsSql.Ef;
 
 public class ClaimServiceDbContext : DbContext, IDataProvider
 {
+  public DbSet<DbClaim> Claims { get; set; }
+  public DbSet<DbChangeArgument> ChangeArguments { get; set; }
+  public DbSet<DbChangeTemplate> ChangeTemplates { get; set; }
+  public DbSet<DbClaimCategory> ClaimCategories { get; set; }
+  public DbSet<DbClaimCategoryExecutor> ClaimCategoryExecutors { get; set; }
+  public DbSet<DbClaimChange> ClaimChanges { get; set; }
+  public DbSet<DbClaimFile> ClaimFiles { get; set; }
+  public DbSet<DbClaimImage> ClaimImages { get; set; }
+  public DbSet<DbComment> Comments { get; set; }
```

Comment:
Maybe "ClaimComments"

Reply:
And db type too. So the naming was consistent.